soil_api/config.py

```diff
@@ -34,59 +34,15 @@ def api_url(self):
         'UNIT["degree",0.0174532925199433, AUTHORITY["EPSG","9122"]], '
         'AUTHORITY["EPSG","4326"]], PROJECTION["Interrupted_Goode_Homolosine"], UNIT["Meter",1]]'
     )
-    soil_types_mapping: dict = {
-        0: "Acrisols",
-        1: "Albeluvisols",
-        2: "Alisols",
-        3: "Andosols",
-        4: "Arenosols",
-        5: "Calcisols",
-        6: "Cambisols",
-        7: "Chernozems",
-        8: "Cryosols",
-        9: "Durisols",
-        10: "Ferralsols",
-        11: "Fluvisols",
-        12: "Gleysols",
-        13: "Gypsisols",
-        14: "Histosols",
-        15: "Kastanozems",
-        16: "Leptosols",
-        17: "Lixisols",
-        18: "Luvisols",
-        19: "Nitisols",
-        20: "Phaeozems",
-        21: "Planosols",
-        22: "Plinthosols",
-        23: "Podzols",
-        24: "Regosols",
-        25: "Solonchaks",
-        26: "Solonetz",
-        27: "Stagnosols",
-        28: "Umbrisols",
-        29: "Vertisols",
-    }
     isric_roi: dict = {
         "min_lat": -90,
         "max_lat": 90,
         "min_lon": -180.0,
         "max_lon": 180,
     }
-    depths: list = ["0-5", "5-15", "15-30", "30-60", "60-100", "100-200"]
-    no_data_val: int = -32768
-    soil_property_to_unit_mapping: dict = {
-        "bdod": "cg/cm³",
-        "cec": "mmol(c)/kg",
-        "cfvo": "cm³/dm³",
-        "clay": "g/kg",
-        "nitrogen": "cg/kg",
-        "ocd": "hg/m³",
-        "ocs": "t/ha",
-        "phh2o": "pH*10",
-        "sand": "g/kg",
-        "silt": "g/kg",
-        "soc": "dg/kg",
-    }
+    no_data_vals_soilgrids: list = [-32768, 65535]
+    no_data_val: int = -99999
+    soil_property_value_types: list = ["mean", "Q0.05", "Q0.5", "Q0.95", "uncertainty"]
```

Comment:
Right now these config/constant values are stored in the `config.py` file. Is this ok? Is there a better place to store them?

Reply:
I would prefer it if config.py is reserved for properties that needs to be configured during deployment via environment variables. Maybe in models/soil_type as an enum?

Reply:
Ok, I've also put some of these values in a separate/new `constants.py` file, is that better?